presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -115,6 +116,11 @@ protected Object computeScalar(@Language("SQL") String sql)
         return computeActual(sql).getOnlyValue();
     }
 
+    protected void assertMatch(String actual, String regex)
```

Comment:
static. It does not belong to AbstractTestQueryFramework, but rather to something like `PrestoAssert`.